src/Ng/Jobs/LoopingNgJob.cs

```diff
@@ -38,7 +38,14 @@ protected LoopingNgJob(ILoggerFactory loggerFactory)
                 }
 
                 Logger.LogInformation("Running job.");
-                await RunInternal(cancellationToken);
+                try
+                {
+                    await RunInternal(cancellationToken);
+                }
+                catch(TransientException te)
```

Comment:
If implemented in the LoopingNgJob all the jobs can only opt-in to use the exception. They will not need to try/catch/handle in their RunInternal.